src/server/pps/cmds/cmds.go

```diff
@@ -205,6 +205,59 @@ $ pachctl list-job -p foo bar/YYY
 			return nil
 		}),
 	}
+	listDatum := &cobra.Command{
+		Use:   "list-datum job-id",
+		Short: "Return the datums in a job.",
+		Long:  "Return the datums in a job.",
+		Run: cmdutil.RunBoundedArgs(1, 1, func(args []string) error {
+			client, err := pachdclient.NewOnUserMachine(metrics, "user")
+			if err != nil {
+				return err
+			}
+			datumInfos, err := client.ListDatum(args[0])
+			if err != nil {
+				return err
+			}
+			if raw {
+				for _, datumInfo := range datumInfos {
+					if err := marshaller.Marshal(os.Stdout, datumInfo); err != nil {
+						return err
+					}
+				}
+			}
+			writer := tabwriter.NewWriter(os.Stdout, 20, 1, 3, ' ', 0)
+			pretty.PrintDatumInfoHeader(writer)
+			for _, datumInfo := range datumInfos {
+				pretty.PrintDatumInfo(writer, datumInfo)
+			}
+			return writer.Flush()
+		}),
+	}
+	inspectDatum := &cobra.Command{
+		Use:   "inspect-datum job-id datum-id",
+		Short: "Display detailed info about a single datum.",
+		Long:  "Display detailed info about a single datum.",
+		Run: cmdutil.RunBoundedArgs(2, 2, func(args []string) error {
+			client, err := pachdclient.NewOnUserMachine(metrics, "user")
+			if err != nil {
+				return err
+			}
+			datumInfo, err := client.InspectDatum(args[0], args[1])
+			if err != nil {
+				return err
+			}
+			if raw {
+				if err := marshaller.Marshal(os.Stdout, datumInfo); err != nil {
+					return err
+				}
+			}
+			writer := tabwriter.NewWriter(os.Stdout, 10, 1, 3, ' ', 0)
+			pretty.PrintDetailedDatumInfo(writer, datumInfo)
+			pretty.PrintDatumPfsStateHeader(writer)
+			pretty.PrintDatumPfsState(writer, datumInfo)
+			return writer.Flush()
+		}),
+	}
```

Comment:
One minor point I just realized, `listDatum` and `inspectDatum` should probably both take the `--raw` flag to just print the datums as json.